lib/rucio/api/replica.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 import datetime
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Any, Iterator, Optional
```

Comment:
`Iterator` should be imported from `collections.abc` (see https://docs.python.org/3/library/typing.html#typing.Iterator)

Reply:
Importing `Iterator` from `collections.abc` now